content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md

```diff
@@ -136,7 +136,7 @@ command. In that case, you should explicitly set `--certificate-renewal=true`.
 
 ## Manual certificate renewal
 
-You can renew your certificates manually at any time with the `kubeadm certs renew` command.
+You can renew your certificates manually at any time with the `kubeadm certs renew` command, with the appropriate command line options.
```

Comment:
we can omit this change, arguably. calling just `kubeadm certs renew` will signal to the user that they need to pick "all" or a specific target.

Reply:
I mean, technically, you could.  But I'd argue that the documentation should make things as easy as possible, to aid new users.

Reply:
i prefer for us not to maintain command output - e.g. a subcommand list in the documentation

Reply:
That makes sense to me.